### PR TITLE
Improve: go to file - helm

### PR DESCRIPTION
**Pull request by stan-gabriel**
This PR...

## Changes
- Add "go to file" for Helm resources

## How to test it
- Open Helm-Charts project
- Go to Helm
- Right-click on a file or hover over the file and click on the "three dots icon"

## Screenshots
image

image

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/navsections/HelmChartSectionBlueprint/HelmChartContextMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import {setLeftMenuSelection} from '@redux/reducers/ui';
+import {isInPreviewModeSelector} from '@redux/selectors';
+
+import {Dots} from '@atoms';
+
+import ContextMenu from '@components/molecules/ContextMenu';
+
+import Colors from '@styles/Colors';
+
+const StyledActionsMenuIconContainer = styled.span<{isSelected: boolean}>`
```
Comment (topliceanurazvan) [inline src/navsections/HelmChartSectionBlueprint/HelmChartContextMenu.tsx:20]: Can we have the styled components in a separate file `.styled` ?

**Review by topliceanurazvan — CHANGES_REQUESTED** [review]
Seems like this is applied only to templates files, can we also have it for values file and helm chart?

File: src/navsections/HelmChartSectionBlueprint/HelmChartContextMenu.tsx (hunk 1 above)
Comment (stan-gabriel) [reply src/navsections/HelmChartSectionBlueprint/HelmChartContextMenu.tsx]: done

**Comment by stan-gabriel** [comment]
@topliceanurazvan there is an issue with the values files so I will open a bug.

File: src/navsections/HelmChartSectionBlueprint/HelmChartContextMenu.tsx [hunk 2]
```diff
@@ -22,11 +22,14 @@ const HelmChartContextMenu: React.FC<ItemCustomComponentProps> = props => {
 
   const dispatch = useAppDispatch();
   const isInPreviewMode = useAppSelector(isInPreviewModeSelector);
+  const resourceMap = useAppSelector(state => state.main.resourceMap);
 
   const onClickShowFile = () => {
+    console.log(itemInstance);
+    console.log(resourceMap);
```
Comment (topliceanurazvan) [inline src/navsections/HelmChartSectionBlueprint/HelmChartContextMenu.tsx:29]: I guess this is not needed anymore

**Review by topliceanurazvan — CHANGES_REQUESTED** [review]

**Comment by devcatalin** [comment]
I will close this for now because it will not be compatible with the section blueprints refactor.
Will try to add this functionality in my PR.

**Pull request closed without merging**